tests/smokescreen/test_compartment_5.py

```diff
@@ -87,22 +88,23 @@ def test_compartment_5(page: Page, smokescreen_properties: dict) -> None:
     SubjectScreeningSummary(page).click_datasets_link()
 
     # Click on 'Show Dataset' next to the Colonoscopy Assessment
+    subject_datasets(page).click_show_datasets()
 
     # Populate Colonoscopy Assessment Details fields
 
     # ASA Grade  - I - Fit
+    subject_datasets(page).select_asa_grade_option("FIT")
+
     # Fit for Colonoscopy (SSP) - Yes
+    subject_datasets(page).select_fit_for_colonoscopy_option("YES")
```

Comment:
The same will need to be done here:
```suggestion
    subject_datasets(page).select_fit_for_colonoscopy_option(FitForColonoscopySspOptions.YES.value)
```